index.css

```
@@ -0,0 +1,210 @@
+@import url('~@noonnu/rokaf-sans-medium'); /*logo 폰트*/
+
+
+* {
+  box-sizing: border-box;
+  margin: 0; /*html안에 margin과 padding 여백 없애주기*/
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Pretendard', sans-serif;
+
+  font-size: 16px;
+}
+
+header {
+  margin: 0 auto;
+  padding: 0 20px;
+  max-width: 1120px;
+  height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.logo-img{
+  width: 40px;
+  margin-right: 8px;
+}
+
+.logo-title {
+  display: inline-block;
+}
+
+.login-btn {
+  display: inline-block;
+  width: 128px;
+  height: 48px;
+  padding: 12px 23px;
+  border-radius: 8px;
+
+  font-size: 1em;
+  font-weight: 500;
+  line-height: 26px;
+  text-align: center;
+  text-decoration: none;
+
+  background-color: rgba(54, 146, 255, 1);
+  color: rgba(243, 244, 246, 1);
+
+  cursor: pointer;
+}
+
+.outside {
+  width: 100%;
+  height: 540px;
+
+  background-color: rgba(207, 229, 255, 1);
+}
+
+
+.outside.top,
+.outside.bottom {
+  display: flex;
+  justify-content: center;
+  align-items: flex-end
+}
+
+
+.top-contents,
+.bottom-contents {
+  display: flex;
+}
+
+
+.top-img,
+.bottom-img {
+  width: 746px;
+  height: 340px;
+}
```

```suggestion
/* 기본 스타일 (모바일) */
.info-container {
  flex-direction: column;
}

/* 태블릿 */
@media (min-width: 768px) {
  .info-container {
    flex-direction: row;
  }
}

/* 데스크톱 */
@media (min-width: 1024px) {
  .info-container {
    padding: 0;
  }
}
```

반응형을 위에처럼 작성해볼 수 있습니다. 일반적으로 위처럼 모바일 first 접근법을 사용합니다. 🙃